en-US/win10/samples/ZigBeeAdapterTutorial.md

```diff
@@ -0,0 +1,203 @@
+
+ZDO command classes such as __ManagementLQI__ class, __ActiveEndPoints__ class… are used to discover the ZigBee network and the ZigBee devices and end points. These classes derive from __ZigBeeCommand__ class.
+
+The __ZclClusterFactory__ class is used to create instances of a specific cluster class. This class is used by the adapter class to create the relevant clusters when it discovers a new ZigBee device. Note that Adapter class create a ZigBeeDevice instance only if that device has at least 1 end point that has a ZCL cluster listed in the supported cluster list of the ZclClusterFactory.
+
+AT command classes such as __AO_Command__,  __HV_Command__…  are used by XBeeModule upon its initialization to get information about the XBee module it uses. These classes derive from the __XBeeATCommand__ abstract class. See XBee ZigBee module documentation (http://www.digi.com) for more detail about AT commands.
+
+### Sending ZDO or ZCL command and receiving response
+
+![SendZdoZcl]({{site.baseurl}}/images/ZigBee/SendZdoZcl.png)
+
+1. When ZclAttribute class reads the value of the attribute, it will build the read attribute payload following the ZCL standard and then call SendCommand method of the ZigBeeCommand class.
+2. SendCommand will then call SendZigBeeCommand method of XBeeModule.
+3. SendZigBeeCommand will build the frame following the format defined by Digi for its XBee module. Once done it will store a reference to the ZigBeeCommand that has been sent in a dictionary and call WriteAsync to send the bytes. After that it will return and SendCommand will wait until it gets a response or time-out.
+4. Upon reception a complete frame from the XBee module, the reception thread of the SerialController will call GetBytesFromModule callback of XBeeModule.
+5. GetBytesFromModule will parse the XBee part of the frame and check if the response received has a matching command. If so if will call the ParseResponse callback of the relevant command and then signal the reception of the response to that command. 
+
+### Receiving ZDO or ZCL command 
+ZigBee device can send ZDO or ZCL command to the XBee module, e.g.: device announce ZDO command which is sent when a device wakes up or join the network, report attribute ZCL command when an attribute of a ZCL cluster is reportable… This kind of command are not sent in response to another command and can be seen as “notification” by ZigBee adapter.  
+
+![ReceiveZdoZcl]({{site.baseurl}}/images/ZigBee/ReceiveZdoZcl.png)
+
+1. Adapter class will build a list of notification it can receive upon initialization. This list contains instance of specific ZigBeeCommand such as DeviceAnnce, ZclReportAttribute.
```

Comment:
Adapter class will build a list of notifications*
This list contains instances*
DeviceAnnce or ZclReportAttribute*?